src/main/keyboard/shortcutHandler.js

```diff
@@ -8,6 +8,7 @@ import { isFile2 } from 'common/filesystem'
 import { isOsx } from '../config'
 import { getKeyboardLanguage, getVirtualLetters } from '../keyboard'
 
+const altOrCtrl = (isOsx) ? 'Ctrl' : 'Alt'
```

Comment:
Please use a better name like `tabSwitchBaseStroke` or similar.

Reply:
Yes on mac crtrl and cmd are different keys. Most apps use slightly different mappings on macos.